app/models/concerns/authorizable.rb

```diff
@@ -16,7 +16,7 @@ module Authorizable
       if user.nil?
         self.where('1=0')
       elsif user.admin?
-        self.scoped
+        self.where(1)
```

Comment:
Tests fail on PostgreSQL as it sees this as a data mismatch.  Shouldn't it be `where(nil)` or `where('1=1')`?